BACKLOG.md

```diff
@@ -0,0 +1,38 @@
+BACKLOG
+=======
+
+General refactoring
+-------------------
+There are 3 representation of a tree.
+
+Remove `LTree`. `LTree` is currently not exported from `muste-lib`, it
+is however used internally. E.g. in `linearizeTree` which is used in
+the ajax-backend.
+
+Style improvements / dead code removal
+-----------------
+
+I consider removing the haste front end and `Main.hs` in `muste-lib`. I
+don't know if this is needed for some special purpose.
```

Comment:
I agree: but after the refactorisation, perhaps consider adding offline tests

Reply:
There is a Test directoy, but it's quite old and obsolete. Perhaps it's easier to scrap it and start over with testing when the refactoring is mostly finished.